src/Stratis.Bitcoin.Features.SmartContracts.Tests/Stratis.Bitcoin.Features.SmartContracts.Tests.csproj

```diff
@@ -118,9 +118,19 @@
   </ItemGroup>
 
   <ItemGroup>
+    <None Update="Packages\1.0.0-TEST\Stratis.SmartContracts.dll">
```

Comment:
So these dlls will be checked in, is this the only way this can be done ?

Reply:
They are part of the test and contain different versions of Stratis.SmartContracts. I guess we could somehow dynamically compile both of them.

Reply:
There is also a dependency on a specific version of System.Runtime, which is why that's included. It's hard/impossible to know if the local machine has that specific version.

Reply:
ok